docs/blog/_posts/2019-03-01-13th-dotty-milestone-release.md

```diff
@@ -0,0 +1,407 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.13.0-RC1
+author: Aggelos Biboudis
+authorImg: /images/aggelos.jpg
+date: 2019-03-01
+---
+
+Hello hello! This is the second release for 2019, let's call it the _Contextual_
+release and you will understand why we are super excited in a bit! ✨🎊🎉
+
+Without further ado, today we release the version 0.13.0-RC1 of the Dotty compiler.
+This release serves as a technology preview that demonstrates new language features and the
+compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
+pioneered the fusion of object-oriented and functional programming in a typed setting. Scala 3 will
+be a big step towards realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs well together,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 132th scheduled release according to our

```

Comment:
132nd

Reply:
Pretty sure we've done less than 132 releases :).

Reply:
Release early, release often!

Reply:
13th 13th 🙈🙈🙈🙈🙈